docs/examples/QuoteInline.jsx

```diff
@@ -0,0 +1,8 @@
+var example = (
+  <div>
+    <p>Gandalf, Captain of the starship enterprise said <UIToolkit.Quote type="inline" author="Gandalf" role="Captain of the Starship Enterprise" cite="The Internet" purpose="success">Use the force Harry</UIToolkit.Quote>
```

Comment:
If I move the `<UIToolkit.Quote>` element to the next line in the docs, the white space between the previous text and the element disappears: it appears to be the case that any amount of white space other than a single space character results in the white space being removed from the rendered result: is this a bug?